### PR TITLE
Allow data from myria and pasted data

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by domoritz** [comment]
Yes, a modal instead of a popup would be better.

File: src/components/addmyriadataset/selectize.default.scss [hunk 1]
```diff
@@ -0,0 +1,442 @@
+/**
```
Comment (kanitw) [inline src/components/addmyriadataset/selectize.default.scss]: I don’t think this file should be here.  

Other directives can use selectize too.  So we better put this in asset?

Comment (domoritz) [inline src/components/addmyriadataset/selectize.default.scss]: Sure, I was just hoping that we only use the css. I'll chnage it.

Comment (kanitw) [inline src/components/addmyriadataset/selectize.default.scss]: No big deal.  :)

**Comment by domoritz** [comment]
I've updated the pr

**Pull request merged**